tests/postgres_tests/test_array.py

```diff
@@ -325,6 +339,33 @@ def test_usage_in_subquery(self):
             [self.objs[3]]
         )
 
+    def test_value_annotate(self):
+        value = models.Value([5], output_field=ArrayField(models.IntegerField()))
+        self.assertEqual(
+            NullableIntegerArrayModel.objects.annotate(value=value).first().value,
+            [5]
+        )
+
+    @unittest.expectedFailure
```

Comment:
I wouldn't add this as we aren't in the habit of adding failing tests like this. It increases the time it takes to run the tests with little benefit.

Reply:
There are already some here and I find them quite helpful, perhaps we could have a decorator that combines `unittest.skip` and `unittest.expectedFailure`? `pytest` has one